src/main/views/search-option.njk

```diff
@@ -22,14 +22,17 @@
                     classes: "govuk-fieldset__legend--l"
                 }
             },
+            hint: {
+              text: "Do you know the name of the court or tribunal?"
```

Comment:
The font colour for this is grey however based on the wireframe I think this should be black.

Reply:
Hi Chris. I have spoken to Nigel about this wireframe. They are wrong. I have added comments on the task about this. They will need to change the wireframe as the gov guideline have the hint text grey and not bold